FEXCore/Source/Interface/Core/X86Tables/X86Tables.h

```diff
@@ -231,6 +231,7 @@ enum InstType {
   TYPE_X87 = TYPE_INST,
   TYPE_INVALID,
   TYPE_COPY_OTHER,
+  TYPE_ARCH_DISPATCHER,
```

Comment:
Should probably have a comment explaining dispatcher behavior (indirection to another table given by OpDispatchPtrWrapper::Indirect) and intended use case (32-/64-bit switching).